app/src/main/res/values/strings.xml

```diff
@@ -555,5 +555,6 @@
     <string name="notify_content">Click to view exported products</string>
     <string name="notification_channel_name">Export Notifications</string>
     <string name="notify_channel_description">Notification to show exported file\'s location</string>
+    <string name="products_to_be_completed">Products to be Completed</string>
```

Comment:
This is tiny but can you remove the capital C from Completed? Just to be more consistent with the rest of the UI. Once that is done, ready for merging.